### DISCUSSION TITLE
leetcode-js/react/11

### DISCUSSION THREAD

**Discussion by giscus** [2xiao/leetcode-js#19] [Announcements]
# leetcode-js/react/11

11. 实现事件系统 摘要 实现 ReactDOM 和 Reconciler 对接 模拟实现浏览器事件流程 相关代码可在 git tag v1.11 查看

https://2xiao.github.io/leetcode-js/react/11.html

**Comment by leonyangdev** [comment]
还算这一节简单一些，前面几节太干了，根本吃不下啊
